migration/migrator/migrations/course/20240219030249_peer_panel_graders.py

```diff
@@ -0,0 +1,34 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    database.execute("ALTER TABLE electronic_gradeable DROP COLUMN IF EXISTS eg_peer_panel;")
```

Comment:
You can remove the first database execute as column doesn't exist on main.

Reply:
So, only the following line will stay ?
`database.execute("ALTER TABLE electronic_gradeable ADD COLUMN eg_peer_panel VARCHAR(5) DEFAULT '11111';")`

But, It also needs to include CREATE TABLE electronic_gradeable  query right?